src/index.js

```diff
@@ -50,9 +50,26 @@ const onfidoRender = (options, el, merge) => {
   return render( <Container options={options}/>, el, merge)
 }
 
+const base64Data = (dataURI) =>
+  ({
+    byteString: atob(dataURI.split(',')[1]),
+    mimeString: dataURI.split(',')[0].split(':')[1].split(';')[0]
```

Comment:
Can you add a comment with an example dataURI.

e.g. `data:image/png;base64,<data>`